modules/audio/src/main/resources/META-INF/substrate/config/jniconfig-x86_64-ios.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
This file is not needed, there are no callbacks from native

Reply:
I removed both jniconfig-arm64-ios.json & jniconfig-x86_64-ios.json files

Is that ok?